app/lib/constants/site_config.rb

```diff
@@ -170,15 +170,15 @@ module SiteConfig
         placeholder: SVG_PLACEHOLDER
       },
       logo_png: {
-        description: "Minimum 1024px, used for PWA etc.",
+        description: "Minimum 192x192px, used for PWA etc.",
```

Comment:
the 1024px has to be a typo right? I'm not sure if there really is a required size at all but chose 192x192px because it's an ideal size to supports [both ios and android](https://stackoverflow.com/questions/48839338/which-icon-sizes-are-required-for-progressive-web-apps-pwa-as-of-q1-2018). Does that sound alright @benhalpern ?